### PR TITLE
removed overcast dependency

### PR DESCRIPTION
**Pull request by EmilDafinov**
Part of the resolution of https://github.com/AppDirect/chatty-pie-connector/issues/81
We have to remove the overcast dependency, since the version we need is not available outside appdirect.

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ spring.datasource:
     #on the `useLegacyDatetimeCode` parameter
     url: jdbc:mysql://localhost/chatty_pie_connector?useLegacyDatetimeCode=false&createDatabaseIfNotExist=true
     username: root
-    password: password
+    password: ""
```
Comment (juan-blasco) [inline src/main/resources/application.yml]: readme file still shows "root/password" for DB setup, does it make sense to update it in this PR too?

Comment (appdirect-opensource) [inline src/main/resources/application.yml]: Definitely

Comment (EmilDafinov) [inline src/main/resources/application.yml]: Definitely

**Comment by EmilDafinov** [comment]
@juan-blasco updated the readme

**Pull request merged**